README.md

```diff
@@ -39,16 +39,24 @@ For more info, visit the tool's [Design Document](https://www.office.com/launch/
 
 # Installation
 ## Manual Installation
-Download this repo to your machine, and then move the Network-Performance-Visualization folder to C:\Program Files\WindowsPowerShell\Modules. Alternatively copy the folder to "%userprofile%\Documents\WindowsPowershell\Modules". After moving the folder, PowerShell will automatically load the module when the commands are invoked. You can double check that everything was installed correctly by running the command
+Download this repo to your machine, and then copy the Network-Performance-Visualization folder to one of the follwing locations:
+- Windows PowerShell (5.1)
+  - `C:\Program Files\WindowsPowerShell\Modules`
```

Comment:
This path doesn't work reliably.  Seems we need to use the one below.

Reply:
There is also a compatibility issue with .NET Core that needs to be fixed for this to work on PS 6.0+ due to https://github.com/PowerShell/PowerShell/issues/11837